src/qt/overviewpage.cpp

```diff
@@ -139,6 +142,15 @@ OverviewPage::OverviewPage(const PlatformStyle *platformStyle, QWidget *parent)
     showOutOfSyncWarning(true);
     connect(ui->labelWalletStatus, &QPushButton::clicked, this, &OverviewPage::handleOutOfSyncWarningClicks);
     connect(ui->labelTransactionsStatus, &QPushButton::clicked, this, &OverviewPage::handleOutOfSyncWarningClicks);
+
+    connect(this, &OverviewPage::setPrivacyMode, ui->labelBalance, &BitcoinAmountLabel::setPrivacyMode);
+    connect(this, &OverviewPage::setPrivacyMode, ui->labelUnconfirmed, &BitcoinAmountLabel::setPrivacyMode);
+    connect(this, &OverviewPage::setPrivacyMode, ui->labelImmature, &BitcoinAmountLabel::setPrivacyMode);
+    connect(this, &OverviewPage::setPrivacyMode, ui->labelTotal, &BitcoinAmountLabel::setPrivacyMode);
+    connect(this, &OverviewPage::setPrivacyMode, ui->labelWatchAvailable, &BitcoinAmountLabel::setPrivacyMode);
+    connect(this, &OverviewPage::setPrivacyMode, ui->labelWatchPending, &BitcoinAmountLabel::setPrivacyMode);
+    connect(this, &OverviewPage::setPrivacyMode, ui->labelWatchImmature, &BitcoinAmountLabel::setPrivacyMode);
+    connect(this, &OverviewPage::setPrivacyMode, ui->labelWatchTotal, &BitcoinAmountLabel::setPrivacyMode);
```

Comment:
I wonder if there's a good way to auto-connect these

Reply:
I believe this could follow this PR.

Reply:
This part of code has been removed in the latest push.